src/components/QueryForm.tsx

```diff
@@ -180,7 +194,50 @@ function QueryForm({
           inputValue={imagingModality}
         />
       </div>
-      <div className="row-start-10">
+      <div className="col-span-2 row-start-9">
+        <CategoricalField
+          label="Pipeline name"
+          options={Object.keys(pipelines).map((p) => ({
+            label: p.slice(3),
```

Comment:
what am I slicing here? 
Ah, I guess `nb:`, yes? 
That's real tricky to remember and needs a comment. 
Additionally we could call `p` -> `pipelineURI` to 
assist with that memory